testing/trino-tests/src/test/java/io/trino/tests/TestGracefulShutdown.java

```diff
@@ -73,9 +73,13 @@ public void testShutdown()
                 .build();
 
         try (DistributedQueryRunner queryRunner = createQueryRunner(TINY_SESSION, properties)) {
-            List<ListenableFuture<?>> queryFutures = new ArrayList<>();
+            List<ListenableFuture<Void>> queryFutures = new ArrayList<>();
             for (int i = 0; i < 5; i++) {
-                queryFutures.add(executor.submit(() -> queryRunner.execute("SELECT COUNT(*), clerk FROM orders GROUP BY clerk")));
+                queryFutures.add(Futures.submit(
+                        () -> {
```

Comment:
Are the curly brackets needed here?  If not remove them, and maybe put the whole thing back on one line

Reply:
Yep, i stared at this for a bit and the curly braces are needed to disambiguate the version that has a return value from the one that does not.